src/main.rs

```diff
@@ -50,9 +54,11 @@ mod version;
 mod vwmap;
 
 fn main() {
+    Builder::new().filter_level(LevelFilter::max()).init();
```

Comment:
if this is needed for `main.rs` (the binary), it's also needed for `lib.rs` (the lib)

Reply:
That's a good point; not sure where in `lib.rs` this fits actually - what do think?

Reply:
there's no perfect place, but the most appropriate one would be in `new_fw_predictor_prototype`, since that is called for every model load (so no large overhead, as opposed to initializing per prediction)

Reply:
[Moved it ](dea63b155d52de6d9896ca43d598694e18b1a2a3)to suggested location - to avoid duplication also added `logging_layer.rs` so that we can just call the init wherever needed + have a single source of config prod/non-prod.